src/dappsExplorer/DappRankingsBottomSheet.tsx

```diff
@@ -16,15 +16,11 @@ export function DappRankingsBottomSheet({
   onPressDapp,
 }: {
   forwardedRef: RefObject<BottomSheetRefType>
-  onPressDapp: (dapp: ActiveDapp) => void
+  onPressDapp: (dapp: ActiveDapp, index: number) => () => void
```

Comment:
Oh, now I understand, but it feels like too much indirection and surprising.
the `onXXX` props are usually just functions.

Can we do differently? and avoid this surprise.